libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
```

Comment:
I'm somehow not happy to have "beat specific" knowledge here but haven't found a better solution yet. I would assume that the below should be true for all message fields but it is not guaranteed. See also TODO on line 100: Perhaps we can find here a better more general way (in a second step).  Any ideas?

As far as I can see this would not have an affect on our beats (right now)?